src/components/Hero/_Hero.scss

```diff
@@ -67,6 +67,39 @@
   }
 }
 
+.hero--tertiary {
+  background-color: var(--ui-gray-xdark);
+
+  .nypl--books-and-more & {
+    background-color: var(--section-books-and-more-primary);
+  }
+
+  .nypl--locations & {
+    background-color: var(--section-locations-primary);
+  }
+
+  .nypl--research & {
+    background-color: var(--section-research-primary);
+  }
+
+  .nypl--whats-on & {
+    background-color: var(--section-whats-on-primary);
+  }
```

Comment:
Let's keep this color organization as is for now, but this should eventually be replaced with a more comprehensive solution for applying background colors.